db-scripts/src/main/resources/migration.sql

```diff
@@ -542,12 +542,12 @@ UPDATE `info` SET `DB_SCHEMA_VERSION`="2.7.2";
 
 ##version: 2.7.3
 DELETE FROM `clinical_attribute_meta` WHERE clinical_attribute_meta.`ATTR_ID` = 'SAMPLE_COUNT';
-INSERT INTO `clinical_attribute_meta` SELECT 'SAMPLE_COUNT', 'Number of Samples Per Patient', 
+INSERT INTO `clinical_attribute_meta` SELECT 'SAMPLE_COUNT', 'Number of Samples Per Patient',
 'Number of Samples Per Patient', 'STRING', 1, '1', patient.`CANCER_STUDY_ID` FROM patient
 GROUP BY patient.`CANCER_STUDY_ID`;
 
 DELETE FROM `clinical_patient` WHERE clinical_patient.`ATTR_ID` = 'SAMPLE_COUNT';
-INSERT INTO `clinical_patient` SELECT patient.`INTERNAL_ID`, 'SAMPLE_COUNT', COUNT(*) FROM sample 
+INSERT INTO `clinical_patient` SELECT patient.`INTERNAL_ID`, 'SAMPLE_COUNT', COUNT(*) FROM sample
```

Comment:
I've been tempted to do the trailing whitespace cleanup myself .. I've looked over the whitespace cleanup you have done here and don't think there would be any functional effect on the earlier migration steps, so I'll approve with these included.